### ISSUE TITLE
[1.3.4] Install fails on python 3.6

### ISSUE THREAD

**Issue by tuxmaster5000** [paver/paver#201]
It will fails with:
Copying Paver.egg-info to /builddir/build/BUILDROOT/python-paver-1.3.4-1.el7.x86_64/usr/lib/python3.6/site-packages/Paver-1.3.4-py3.6.egg-info
running install_scripts
Installing paver script to /builddir/build/BUILDROOT/python-paver-1.3.4-1.el7.x86_64/usr/bin
  File "/usr/lib/python3.6/site-packages/paver/deps/path2.py", line 939
    def mkdir(self, mode=0777):

**Comment by Almad** [comment]
@tuxmaster5000 could you please post your OS/version and install process? 

(package? pip install? system-wide or virtualenv? root or user?)

**Comment by tuxmaster5000** [comment]
It will be an system-wide build for rpm. Using CentOS 7.6
I have add the build log here:
[build.txt](https://github.com/paver/paver/files/3329177/build.txt)

**Comment by Almad** [comment]
I see the problem, for some reason the build is executing all files, attempting to bundle in python2 file with python3 interpreter. Apparently you can't bytecompile without parsing ;)

Can you convince `/usr/lib/rpm/brp-python-bytecompile` to skip this file, use a different interpreter or just remove that file before building the rpm package for py3?

**Comment by tuxmaster5000** [comment]
No, because the build script from CentOS/RedHat will precompile all installed python files.
As an temporary workaround  I have remove the file, before I call setup.py. But it will be better, when the setup only install the python2 stuff, when it called by python2.

**Comment by Almad** [comment]
Do you know how to do that?

**Comment by tuxmaster5000** [comment]
I have tried to understood your installer, but it looks for me very strange, because it don't use the default setup.py routines.
